modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -213,6 +213,16 @@ public IgniteToKafkaCdcStreamer(
 
     /** {@inheritDoc} */
     @Override public void start(MetricRegistry mreg) {
+        if (kafkaProps == null)
```

Comment:
Let's add A.notNull(topic) here.

Reply:
Done.

I have done the symmetric check for KafkaToIgniteStreamer too.